src/i18n/pl/nav.ts

```diff
@@ -0,0 +1,125 @@
+/**
+ * This configures the navigation sidebar.
+ * All other languages follow this ordering/structure and will fall back to
+ * English for any entries they haven’t translated.
+ *
+ * - All entries MUST include `text` and `key`
+ * - Heading entries MUST include `header: true` and `type`
+ * - Link entries MUST include `slug` (which excludes the language code)
+ */
+export default [
+	{ text: 'Zacznij tutaj', header: true, type: 'learn', key: 'zacznijTutaj' },
```

Comment:
The `nav.ts` file for other languages that aren't English follow a different format, take a look at the PT-BR one: 
https://github.com/withastro/docs/blob/main/src/i18n/pt-br/nav.ts

Changing this should fix the CI.